[2023-12-20 00:36:41,964] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL.conexion_BD scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2023-12-20 00:36:41,998] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL.conexion_BD scheduled__2023-12-19T00:00:00+00:00 [queued]>
[2023-12-20 00:36:41,999] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-20 00:36:42,001] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-20 00:36:42,002] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-20 00:36:42,076] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-12-19 00:00:00+00:00
[2023-12-20 00:36:42,098] {standard_task_runner.py:52} INFO - Started process 198 to run task
[2023-12-20 00:36:42,142] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL', 'conexion_BD', 'scheduled__2023-12-19T00:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather.py', '--cfg-path', '/tmp/tmpbbaespd2', '--error-file', '/tmp/tmp48qebwwc']
[2023-12-20 00:36:42,151] {standard_task_runner.py:80} INFO - Job 60: Subtask conexion_BD
[2023-12-20 00:36:43,104] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL.conexion_BD scheduled__2023-12-19T00:00:00+00:00 [running]> on host 44a36bef83ad
[2023-12-20 00:36:43,596] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-20 00:36:43,986] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-12-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-19T00:00:00+00:00
[2023-12-20 00:36:44,007] {logging_mixin.py:115} INFO - Conectandose a la DB en la fecha: 2023-12-19 0
[2023-12-20 00:36:45,335] {logging_mixin.py:115} INFO - <connection object at 0x7f3e1fa48ad0; dsn: 'user=juliocfurtado93_coderhouse password=xxx dbname=data-engineer-database host=data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com port=5439', closed: 0>
[2023-12-20 00:36:45,336] {logging_mixin.py:115} INFO - Conexion a RedShift exitosa!
[2023-12-20 00:36:45,340] {python.py:173} INFO - Done. Returned value was: None
[2023-12-20 00:36:45,459] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL, task_id=conexion_BD, execution_date=20231219T000000, start_date=20231220T003641, end_date=20231220T003645
[2023-12-20 00:36:45,649] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-20 00:36:45,932] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
